src/gps.cpp

```diff
@@ -777,22 +858,32 @@ int32_t Gps::getMessagesWithFailedCrcCount() const {
 }
 
 void Gps::showWaitStatus(DisplayDevice const * display) const {
-  String satellitesString[2];
+  static bool clear = false;
+  if (!is_neo6() && !clear) {
```

Comment:
Usually it is printed as next line in the obs init log - you can see other (not GPS related) init messages before the GPS status. 

I'm also fine if we use the whole display for GPS status - once everything before went fine. I would suggest to use the same code and display for both modules.

Reply:
Neo6m doesn't give us some of the values that we're displaying, as that's a thing starting from 7m or 8m. So either we have users questioning why they see a constant zero (or even noise...?) for @Jens-Kassel's new values gain and jamming we will still need separat codepaths for 8m and 6m. Also I didn't want to pollute the screen for the normally well-working neo6m.

Reply:
Okay ;)